doc/sailfish-webview-webengine.qdoc

```diff
@@ -0,0 +1,326 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/*!
+    \qmltype WebEngine
+    \inherits Item
+    \inqmlmodule Sailfish.WebEngine
+    \ingroup sailfishos-webview
+    \brief Provides access to the Web engine context.
+
+    The \c{WebEngine} component is a singleton which provides access to the Web engine context.
+
+    The \c{WebEngine} QML type is exposed through the \c{Sailfish.WebEngine} QML
+    import namespace.
+
+    Exaple of use:
+
+    \qml
+        import QtQuick 2.0
+        import Sailfish.Silica 1.0
+        import Sailfish.WebView 1.0
+        import Sailfish.WebEngine 1.0
+
+        ApplicationWindow {
+            id: root
+            initialPage: Component {
+                WebViewPage {
+                    id: page
+                    WebView {
+                        anchors.fill: parent
+                        active: true
+                        url: "http://www.sailfishos.org"
+                    }
+
+                    Component.onCompleted: {
+                        WebEngine.addObserver("clipboard:setdata")
+                        WebEngine.onRecvObserve.connect(function(message, data) {
+                            if (message == "clipboard:setdata" && !data.private) {
+                                console.log("Clipboard contents: " + data.data);
+                            }
+                        })
+                    }
+                }
+            }
+        }
+    \endqml
+
+    \sa WebView
+*/
+
+/*!
+    \readonly
+    \qmlproperty bool WebEngine::initialized
+    \brief Returns true if the component has been initialized, false otherwise.
+*/
+
+/*!
```

Comment:
Hrm, the explicit duplication of documentation for C++ and QML sides is ... unfortunate.  However, I don't have any suggestion for how to resolve it..

Reply:
I'm happy to act on any suggestions, but I don't have a solution either I'm afraid.